data/remnant missions.txt

```diff
@@ -655,149 +655,288 @@ mission "Remnant: Learn Sign Follow Up"
+			label nothelp
+			`	Taely looks at you with a blank expression, then chants in a monotone voice "If that is your choice, you are unlikely to find much of interest to do among us." With that, she vaults over the edge of the railing and drops from view. As you quickly peer over the railing, you discover that she had must have caught a handhold somehow, as she is rapidly moving down the wall in a controlled fashion with ease of a spider.`
+				decline
+	to complete
+		or
+			has "Remnant: Heavy Laser: done"
+			has "Remnant: Catalytic Ramscoop: done"
+			has "Remnant: Plasma Cannons: done"
+			has "Remnant: Electron Beam: done"
+			has "Remnant: D94-YV Shield Generator: done"
+			has "Remnant: S-970 Regenerator: done"
+
+
+
 mission "Remnant: Heavy Laser"
 	name "Retrieve Heavy Lasers for the Remnant"
-	description "A Remnant engineer on <planet> has offered to pay you <payment> in exchange for obtaining two heavy lasers for them to study."
+	description "A Remnant engineer has offered to pay you <payment> in return for delivering two heavy lasers to <planet> for them to study."
 	minor
-	source
+	source "Viminal"
+	destination
 		government "Remnant"
 	to offer
-		has "Remnant: Technology Available: offered"
-		not "Remnant: Catalytic Ramscoop: active"
-		not "Remnant: Catalytic Ramscoop: done"
-		not "Remnant: Plasma Cannons: active"
-		not "Remnant: Plasma Cannons: done"
-		random < 50
+		or
+			has "Remnant: Tech Retrieval: active"
+			has "Remnant: Tech Retrieval: done"
 	on offer
 		require "Heavy Laser"
 		conversation
-			`As you are browsing the spaceport shops, a woman approaches you and introduces herself as a researcher at one of the Remnant weapons facilities. She awkwardly pauses before beginning to recite a quick tune: "We can see that your ship has beam weapons that match human techniques, but are unlike anything we have record of. What kind of weapons are they?"`
+			`You remember that Taely was interested in seeing examples of technology from human space. Would you like to show her your heavy laser cannons?`
 			choice
-				`	"They're heavy lasers."`
-				`	"I'd rather not share that."`
-					decline
-			
-			`	"Really? Our records indicate that humanity had primitive laser technology at the time of the Exodus, but nothing that would be considered useful as weapons." She pauses to look down at her scanner. "And yet our scans indicate that they are capable of significant power output.`
-			`	"Based on these scans, we may well be able to learn quite a bit from these weapons. Would you be able to acquire a pair of them for us? We could offer you very good compensation for doing so."`
-			choice
-				`	"Sure, I would be glad to accept that deal."`
-				`	"Sorry, I'm not really interested in helping Remnant research."`
-					decline
-
-			`	After a bit of haggling, she agrees to pay you <payment> in exchange for two heavy lasers.`
+				`	Not now.`
+					defer
+				`	Yes.`
+			branch stuff
```

Comment:
How about "done" instead of "stuff", as this is checking if a prior mission was done. Same with the rest.

Reply:
I ended up using "familiar" since it was indicating if the player was a bit more familiar with Taely. Changed throughout.